apps/combine-service/src/spec/spec.yml

```diff
@@ -417,6 +417,9 @@ paths:
                       master: true
                   files:
                   - <FILE>
+            encoding:
```

Comment:
@bilalshaikh42 in my experience, this needed to be added back. Please review.

Reply:
I was having some trouble with generating a client library with that field present. What is the reason it is needed? I am not familiar with the function of the "encoding" field.

Reply:
I see the issue. i did not realize that the encoding was needed for the input validation.